smarts/core/default_map_builder.py

```diff
@@ -58,23 +59,19 @@ def get_road_map(map_spec) -> Tuple[RoadMap, str]:
         _existing_map = None
         gc.collect()
 
-    map_path = map_spec.source
-    if not os.path.isfile(map_path):
+    if not os.path.isfile(map_spec.source):
         map_path = os.path.join(map_spec.source, "map.net.xml")
```

Comment:
Isn't there a need to check if the offset were normalized and the shifted network file needs to be used here since we create the road map class using this file?

Reply:
Not here.  The shifting of a the map is just a SUMO-specific thing... we try to encapsulate it and the shifted map's name entirely in the `SumoRoadNetwork` class.  However here, in `default_map_builder.py`, we are trying to determine _which_ type of map the scenario uses, in general (SUMO or OpenDrive), so `map_spec.source` is just a folder, but there is presence of a `map.net.xml` file, which is the default Sumo road network filename, this is indicative that we are using Sumo and hence we can proceed.  

Note that since this hasn't yet been merged with the OpenDrive PR, all the current implementation is doing is just trying to determine if this scenario indeed uses a Sumo road network or not.  Even if we ultimately shift the map in `sumo_road_network.py`, there sill should always be a `map.net.xml` file in the folder if there wasn't a network file named explicitly in the `map_spec`.

Reply:
Yup, this makes it easy for OpenDRIVE extension since we only have one network file here.